webapp/channels/src/components/team_sidebar/team_sidebar.tsx

```diff
@@ -286,6 +289,7 @@ export class TeamSidebar extends React.PureComponent<Props, State> {
                         }
                         content={plusIcon}
                         switchTeam={this.props.actions.switchTeam}
+                        displayName='Create a Team'
```

Comment:
This line also needs to be updated to use `intl.formatMessage` as well. Typically, any text in the app that's visible to the user or available to them via accessibility tools needs to be translatable